src/App.vue

```diff
@@ -214,6 +214,14 @@ export default class App extends Mixins(
       sessionStorage.removeItem('NR_DATA')
     }
 
+    // if there is stored legal type for an IA then incorporate it now
+    const legaltype = sessionStorage.getItem('LEGAL_TYPE')
+    if (legaltype) {
+      await this.incorporateNow(legaltype)
+      // clear the legal type data
+      sessionStorage.removeItem('LEGAL_TYPE')
+    }
+
```

Comment:
Check also Signin.vue as it may need similar code.

Reply:
I tested this extensively Sev. Apparently, the code in Signin.vue is not being executed when I go with this flow. So, I only added the code to App.vue. Not sure if I'm missing a case in which the code in Signin.vue would run.

Reply:
What scenarios did you test?

Did you put a breakpoint (or console.log) in Signin.vue to see when it's used?

It's possible that code in Signin.vue is a "decoy" (looks like it's doing something but really isn't) but I'm not sure.

Reply:
Here is what causes Signin.vue to be rendered:
https://github.com/bcgov/namerequest/blob/92172b3bf0756722475bf804249b0979ed624664/src/router/routes.ts#L25

Reply:
Resolved.